docker/publish.sh

```diff
@@ -62,11 +62,11 @@ set -x
 # Fail if any of the subsequent commands fail
 set -e
 
-docker tag pulsar:latest ${docker_registry_org}/pulsar:latest
-docker tag pulsar-all:latest ${docker_registry_org}/pulsar-all:latest
+docker tag apachepulsar/pulsar:latest ${docker_registry_org}/pulsar:latest
+docker tag apachepulsar/pulsar-all:latest ${docker_registry_org}/pulsar-all:latest
```

Comment:
Do you know if this will override "latest" when we release older versions? For example, when we release 3.1.0, we wouldn't want some 3.0.x release to overwrite "latest" in my opinion.

Reply:
Thanks for sharing that reference. I reopened and commented on the issue.